src/main/java/de/relimit/commons/markdown/blockelement/BlockElementNodeBuilder.java

```diff
@@ -83,21 +85,15 @@ protected BlockElement gateKeep(BlockElement element) {
 	// Lists
 
 	public OrderedListBuilder<B> startOrderedList() {
-		final int indentationLevel = getElement().getIndentationLevel();
-		final OrderedList list = new OrderedList(indentationLevel);
-		return new OrderedListBuilder<>(list, this::append);
+		return new OrderedListBuilder<>(new OrderedList(), this::append);
```

Comment:
Just for confirmation: Are these changes correct? Or are the three removed lines the new changes and the supposed changed one line was the old one? (I am thinking about the merge conflict before)

Reply:
They are correct.

I got confused by how the indentation level worked. When I first implemented this I had in mind that every element was responsible for all the indentation. So I assumed with nodes starting at 0 (no indentation) a child on level 5 would add 5 times the indentation (5 x four spaces). However, this paradigm changed during development without me noticing.

Every parent element in fact creates an indentation which is automatically added to every line of child element output. In other words children only need to add 1 (one) indentation if they need to be further indented in relation to their parent. Or noting if they need to remain on their parent's level.

This means that the child indentation level ist always hardcoded to either 0 (zero) or 1.

It is possible to set an indentation of say 3 on any node. E.g. the `Document` which is often the root node. This works: The whole document is moved from level 0 to level 3 (3 x four spaces) and the whole tree moves with it. It is questionable if that makes sense but I left the possibility to increase the indentation in the code.

If however an indentation level is set on any node, the indentation levels of all children are reset to default (0 / 1). I am not sure if that makes sense. But I did not want to change that for now.